### DISCUSSION TITLE
[ 199.048596] virtio_rpmsg_bus virtio0: msg received with no recipient

### DISCUSSION THREAD

**Discussion by Minerro886** [OpenAMP/open-amp#447] [Q&A]
Because I have the need to send multiple rounds of information to the rpu, I added a loop to achieve the corresponding function. But I don't know why, I can send messages successfully for the first time, but I can no longer send messages to rpu later, and the following error occurs. What is the reason. My platform is zcu102 from Xilinx

[ 199.048596] virtio_rpmsg_bus virtio0: msg received with no recipient

**Comment by arnopo** [comment]
Hello @Minerro886 
I converted in discussion as it is not an issue in the openamp library, but only a request of some help to debug.
 
The trace you mention is a Linux kernel trace. So first, have a look to related trace in the Linux kernel code. You will see that your issue is probably that you send a rpmsg to the Linux with an unknown destination address. 
Activating debug trace in virtio_rpmsg_bus.c would allow you to see the contents of the rpmsg, very useful for debugging the IPC.
